ci/test_plugins.rb

```diff
@@ -115,8 +141,8 @@ def install_gem(gem_file)
   :tier1 => {
```

Comment:
in the plugin section, instead of removing the pipelines, can we please comment them out and add a comment that states why, and a reference to the issue? e.g.:

:input => [
                  # "logstash-input-couchdb_changes",  # Skipped because of issue #XXXYYY
                  "logstash-input-gelf"
                  
                  
Lets do this for all plugins being skipped.

Also, hard to see in the diff, is logstash-integration-logstash being covered here? if not, we can file a separate issue to add it.